### DISCUSSION TITLE
How to find value ? help me pls

### DISCUSSION THREAD

**Discussion by WagHaLangThang** [community/community#114391] [Programming Help]
### Body

https://github.com/JambonCru/Logitech-CS-GO-NoRecoil/blob/master/csgo_norecoil_script.lua

I accidentally found this LUA script and tried to create a LUA script with the GetMousePosition() statement to determine the mouse position when shooting AK in CSGO2, but it only returns 1 result: the first position when clicking the mouse even though has moved the mouse pointer to another point. How can they accurately determine the path of the mouse to use the MoveMouseRelative() command to counteract the gun's recoil?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by SiGMafEAr** [comment]
maybe this help you:
https://github.com/doctorwwho/Logitech-Mouse-Anti-Recoil-Script/blob/master/ALL.lua
